client/src/components/Bills/BillList.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import BillForm from "./BillForm";
+
+const emptyBill = {
```

Comment:
❗ this is duplicated from `BillForm` - you could export from the bill form:

```

export default BillForm;
export { emptyBill };

```


```
import BillForm { emptyBill } from "./BillForm";
```

Reply:
Turns out I only use these in the Forms, but I went ahead and exported those consts for contacts, bills, and debts